src/main/jhipster/src/main/webapp/app/germplasm-details/genotyping/genotyping-pane.component.ts

```diff
@@ -169,16 +172,24 @@ export class GenotypingPaneComponent implements OnInit {
     }
 
     selectVariantsetOnChange() {
-        this.genotypingBrapiService.searchCallsets({
-            variantSetDbIds: [this.selectedVariantSet.variantSetDbId],
-            germplasmDbIds: [this.genotypingGermplasm.germplasmDbId]
-        }).subscribe((brapiResponse) => {
-            if (brapiResponse && brapiResponse.result.data.length) {
-                this.genotypingCallSet = brapiResponse.result.data[0];
-                this.loadGenotypingCalls();
-            } else {
-                this.alertService.error('genotyping.no.genotyping.callsets.found');
-            }
+        const exportFlapjackRequest = new ExportFlapjackRequest([], [], 'FLAPJACK', [this.germplasmSearchValue], true,
+            100, this.selectedVariantSet.referenceSetDbId);
+        this.genotypingBrapiService.exportFlapjack(exportFlapjackRequest).subscribe((response) => {
+            let file = response.replace('.fjzip', '').replace('/gigwaV2', '');
+            file = this.cropGenotypingParameter.baseUrl + file;
+
+            const renderer = flapjack.default();
+            renderer.renderGenotypesUrl({
+                domParent: '#flapjack-div',
+                width: 750,
+                height: 300,
+                mapFileURL: file + '.map',
+                genotypeFileURL: file + '.genotype',
```

Comment:
@mderamos Make the flapjack-bytes component's width dynamic, so that when the germplasm details is opened in a new tab, flapjack will also adjust its width relative to its container.

  const flapjackDiv = '#flapjack-div';
            const renderer = flapjack.default();
            renderer.renderGenotypesUrl({
                domParent: flapjackDiv,
                width: document.querySelector(flapjackDiv).getBoundingClientRect().width,
                height: 300,
                mapFileURL: file + '.map',
                genotypeFileURL: file + '.genotype',
                phenotypeFileURL: file + '.phenotype',
                overviewWidth: document.querySelector(flapjackDiv).getBoundingClientRect().width,
                overviewHeight: 35,
                dataSetId: this.cropGenotypingParameter.programId,
            });